.rubocop.yml

```diff
@@ -6,7 +6,7 @@ AllCops:
   - "*/bin/**/*"
   - "**/tmp/**/*"
   - "*/spec/fixtures/**/*"
-  TargetRubyVersion: 2.7
+  TargetRubyVersion: 3.1
```

Comment:
If we had a `.ruby-version` file, this wouldn't be necessary: https://docs.rubocop.org/rubocop/configuration.html#setting-the-target-ruby-version